docs/elements/guides/sign-in.mdx

```diff
@@ -0,0 +1,267 @@
+            <Clerk.Label>Email code</Clerk.Label>
+            <Clerk.Input />
+            <Clerk.FieldError />
+          </Clerk.Field>
+
+          <SignIn.Action submit>Continue</SignIn.Action>
+        </SignIn.Strategy>
+
+        <SignIn.Strategy name="password">
+          <h1>Enter your password</h1>
+
+          <Clerk.Field name="password">
+            <Clerk.Label>Password</Clerk.Label>
+            <Clerk.Input />
+            <Clerk.FieldError />
+          </Clerk.Field>
+
+          <SignIn.Action submit>Continue</SignIn.Action>
+          <SignIn.Action navigate="forgot-password">
+            Forgot password?
+          </SignIn.Action>
+        </SignIn.Strategy>
+
+        <SignIn.Strategy name="reset_password_email_code">
+          <h1>Check your email</h1>
+          <p>We sent a code to <SignIn.SafeIdentifier />.</p>
+
+          <Clerk.Field name="code">
+            <Clerk.Label>Email code</Clerk.Label>
+            <Clerk.Input />
+            <Clerk.FieldError />
+          </Clerk.Field>
+
+          <SignIn.Action submit>Continue</SignIn.Action>
+        </SignIn.Strategy>
+      </SignIn.Step>
+
+      <SignIn.Step name="forgot-password">
+        <h1>Forgot your password?</h1>
+
+        <SignIn.SupportedStrategy name="reset_password_email_code">
+          Reset password
+        </SignIn.SupportedStrategy>
+
+        <SignIn.Action navigate="back">Go back</SignIn.Action>
+      </SignIn.Step>
+
+      <SignIn.Step name="reset-password">
+        <h1>Reset your password</h1>
+
+        <Clerk.Field name="password">
+          <Clerk.Label>New password</Clerk.Label>
+          <Clerk.Input />
+          <Clerk.FieldError />
+        </Clerk.Field>
+
+        <Clerk.Field name="confirmPassword">
+          <Clerk.Label>Confirm password</Clerk.Label>
+          <Clerk.Input />
+          <Clerk.FieldError />
+        </Clerk.Field>
+
+        <SignIn.Action submit>Reset password</SignIn.Action>
+      </SignIn.Step>
+    </SignIn.Root>
+  )
+}
+```
+
+To enable users to reset their passwords, two additional steps, `forgot-password` and `reset-password`, can be added. The `forgot-password` step initiates the reset process, whereby an email code is sent to the user for verification. Once verified, the `reset-password` step allows the user to input a new password. If your instance has been set up to accept SMS codes, the `reset_password_phone_code` strategy can also be used.
+
+The `forgot-password` step renders a new component, [`<SignIn.SupportedStrategy>`](/docs/elements/reference/sign-in#supportedstrategy). This component is also used in the `forgot-password` and `choose-strategy` steps to trigger the verification of a supported strategy. In this context, it triggers the verification of one of the reset_password strategies.
+
+<Callout>
+  If your instance isn't configured to use passwords, or any of the strategies outlined here, Clerk Elements will log a warning to the console during development.
```

Comment:
```suggestion
  If your instance isn't configured to use any of the strategies outlined here, Clerk Elements will log a warning to the console for development instances.
```
I think this is more clear, but is my change accurate? If no we should reword so it's clearer what "in development" means

Reply:
"In development" here means when `NODE_ENV` is set to `development`. While that information is correct, it's not really helpful for the end user. In reality, it means: When you run the development server of your framework, e.g. `next dev`